lib/modules/pageNavigator.js

```diff
@@ -65,7 +65,7 @@ function backToTop() {
 
 function backToNewCommentArea() {
 	const commentArea = document.querySelector('.commentarea > form.usertext textarea');
-	if (!commentArea) return;
+	if (!commentArea || !commentArea.offsetParent) return;
 	const $backToNewCommentArea = $(`<a class="pageNavigator res-icon" data-id="addComment" href="#comments" title="${i18n('pageNavToCommentTitle')}">&#xF003;</a>`);
```

Comment:
Oh right, it's rendered so it can be cloned for replies to comments but hidden initially.